nixpkgs_review/cli/comments.py

```diff
@@ -0,0 +1,183 @@
+import argparse
+
+from ..github import GithubClient
+from .utils import ensure_github_token, get_current_pr
+
+from dataclasses import dataclass, field
+from datetime import datetime
+from typing import List, Union, Optional, Dict, Any
+
+
+def comments_query(pr: int) -> str:
+    return """
+{
+    repository(owner: "NixOS", name: "nixpkgs") {
+        pullRequest(number: %d) {
+            author { login }
+            body
+            createdAt
+            comments(last: 50) {
+               nodes {
+                   author { login }
+                   body
+                   createdAt
+               }
+               totalCount
+            }
+            reviews(last: 50) {
+                totalCount
```

Comment:
In future it might be good to handle cases where more than 50 comments or reviews have been made.